presto-orc/src/main/java/com/facebook/presto/orc/cache/StorageOrcFileTailSource.java

```diff
@@ -152,4 +185,21 @@ private static void checkOrcVersion(OrcDataSource orcDataSource, List<Integer> v
             }
         }
     }
+
+    /**
+     * ORC metadata and DWRF stripe cache sizes are mutually exclusive because
+     * only ORC files have metadata, and only DWRF files have stripe cache.
+     * <p>
+     * Let's check that either both sizes are 0, or only one of them is
+     * greater than 0.
+     */
+    private static void checkSizes(OrcDataSource orcDataSource, int metadataSize, int dwrfStripeCacheReadSize)
```

Comment:
name the dwrfStripeCacheReadSize as dwrfStripeCacheSize.

Reply:
done